numpy/core/numeric.py

```diff
@@ -864,6 +862,7 @@ def correlate(a, v, mode='valid', old_behavior=False):
     See Also
     --------
     convolve : Discrete, linear convolution of two one-dimensional sequences.
+    multiarray.correlate : Old, no conjugate, version of correlate.
```

Comment:
``multiarray`` isn't a public submodule, so this isn't right. Either we add ``np.cross_correlate`` or we just don't offer this behavior at all anymore.